app/src/main/java/com/kickstarter/viewmodels/CommentsViewModel.kt

```diff
@@ -25,18 +28,22 @@ interface CommentsViewModel {
         fun currentUserAvatar(): Observable<String?>
         fun enableCommentComposer(): Observable<Boolean>
         fun showCommentComposer(): Observable<Void>
+        fun commentsList(): Observable<List<Comment>?>
```

Comment:
Does it needs to be optional? in case of no comments we could have an empty list so no null allowed on this Observable

Reply:
Changed